tests/ai_hub/model_registry/rest_api/conftest.py

```
@@ -452,9 +467,10 @@ def model_registry_connection_secret(
     with the opendatahub.io/connections annotation.
     """
     resource_name = "mr-test-inference-service-connection"
-    # Use the model URI from the registered model
+    # Use the model URI from the registered model artifact
     register_model_data = registered_model_rest_api.get("register_model", {})
```

Flake8 (7.3.0)

[error] local variable 'register_model_data' is assigned to but never used

(F841)